.env

```
@@ -2,31 +2,49 @@
-TOP_DIMENSIONS_FOR_ANOMALY_DRILLDOWN=10  # Sets the maximum number of dimensions shown in the Anomaly Drill Downs
-MIN_DATA_IN_SUBGROUP=30  # The minimum population in a subgroup.
-TOP_SUBDIMENSIONS_FOR_ANOMALY=10  # Sets the maximum number of sud-dimensions shown in the Anomaly Sub-dimensions page.
-MAX_FILTER_SUBGROUPS_ANOMALY=250  # Sets the maximum number of subgroups considered for Anomaly Detection
-MAX_ANOMALY_SLACK_DAYS=14  # Sets the maximum number of days for which we can have no data and still consider the KPI for Anomaly Detection. 
+# Enables the generation of multi-dimensional subgroups.
+MULTIDIM_ANALYSIS_FOR_ANOMALY=False
+# Sets the maximum number of unique values allowed in a dimension.
+MAX_SUBDIM_CARDINALITY=1000
+# Sets the maximum number of dimensions shown in the Anomaly Drill Downs.
+TOP_DIMENSIONS_FOR_ANOMALY_DRILLDOWN=10
+# The minimum population in a subgroup.
+MIN_DATA_IN_SUBGROUP=30
+# Sets the maximum number of sud-dimensions shown in the Anomaly Sub-dimensions page.
+TOP_SUBDIMENSIONS_FOR_ANOMALY=10
+# Sets the maximum number of subgroups considered for Anomaly Detection
+MAX_FILTER_SUBGROUPS_ANOMALY=250
+# Sets the maximum number of days for which we can have no data and still consider the KPI for Anomaly Detection.
+MAX_ANOMALY_SLACK_DAYS=14
 
 ### DeepDrills Configuration
-MAX_ROWS_FOR_DEEPDRILLS=10000000  #Sets the maximum number of rows allowed for a KPI to be added.
-MAX_DEEPDRILLS_SLACK_DAYS=14  # Sets the maximum number of days for which we can have no data and still consider the KPI for DeepDrills.
-DEEPDRILLS_HTABLE_MAX_PARENTS=5  # Sets the maximum number of rows in the first level of the DeepDrills' drilldowns.
-DEEPDRILLS_HTABLE_MAX_CHILDREN=5  # Sets the maximum number of rows in the subsequent levels of the DeepDrills' drilldowns.
-DEEPDRILLS_HTABLE_MAX_DEPTH=3  # Sets the maximum depth of the drilldowns in DeepDrills.
-DEEPDRILLS_ENABLED_TIME_RANGES=last_30_days,last_7_days,previous_day,month_on_month,month_to_date,week_on_week,week_to_date  # Sets the enabled time ranges for which DeepDrills is computed as comma separated values.
+# Sets the maximum number of rows allowed for a KPI to be added.
+MAX_ROWS_FOR_DEEPDRILLS=10000000
+# Sets the maximum number of days for which we can have no data and still consider the KPI for DeepDrills.
+MAX_DEEPDRILLS_SLACK_DAYS=14
+# Sets the maximum number of rows in the first level of the DeepDrills' drilldowns.
+DEEPDRILLS_HTABLE_MAX_PARENTS=5
+# Sets the maximum number of rows in the subsequent levels of the DeepDrills' drilldowns.
+DEEPDRILLS_HTABLE_MAX_CHILDREN=5
+# Sets the maximum depth of the drilldowns in DeepDrills.
+DEEPDRILLS_HTABLE_MAX_DEPTH=3
+# Sets the enabled time ranges for which DeepDrills is computed as comma separated values.
+DEEPDRILLS_ENABLED_TIME_RANGES=last_30_days,last_7_days,previous_day,month_on_month,month_to_date,week_on_week,week_to_date  
```

Trailing spaces here too
```suggestion
DEEPDRILLS_ENABLED_TIME_RANGES=last_30_days,last_7_days,previous_day,month_on_month,month_to_date,week_on_week,week_to_date
```